### PR TITLE
Update project status

### PR DESCRIPTION
**Pull request by adamwalach**
This PR is a 
 - [ ] Feature Implementation
 - [ ] Bug Fix
 - [x] Documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** 
https://github.com/kyma-project/kyma/issues/5217

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,17 +34,15 @@ Open Service Broker API project's [Getting Started guide](https://github.com/ope
 
 ### Project Status
 
-We are currently working toward a beta-quality release to be used in conjunction with
-Kubernetes 1.9. See the
-[milestones list](https://github.com/kubernetes-sigs/service-catalog/milestones?direction=desc&sort=due_date&state=open)
+Service Catalog recently switched to a new [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources) based architecture. The old API-server based implementation is available on the [v0.2.0 branch](https://github.com/kubernetes-sigs/service-catalog/tree/v0.2.0). We plan to support this implementation (by providing bug fixes) till July 2020.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
Service Catalog recently switched to a new [CRDs-based](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources) architecture. The old API Server-based implementation is available on the [v0.2.0 branch](https://github.com/kubernetes-sigs/service-catalog/tree/v0.2.0). We plan to support this implementation by providing bug fixes until July 2020.
```

File: README.md [hunk 2]
```diff
@@ -34,17 +34,15 @@ Open Service Broker API project's [Getting Started guide](https://github.com/ope
...
+
+We are currently working toward a beta-quality release. See the [milestones list](https://github.com/kubernetes-sigs/service-catalog/milestones?direction=desc&sort=due_date&state=open)
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
We are currently working towards a beta-quality release. See the [milestones list](https://github.com/kubernetes-sigs/service-catalog/milestones?direction=desc&sort=due_date&state=open)
```

File: README.md [hunk 3]
```diff
@@ -34,17 +34,15 @@ Open Service Broker API project's [Getting Started guide](https://github.com/ope
...
 for information about the issues and PRs in current and future milestones.
 
 The project [roadmap](https://github.com/kubernetes-sigs/service-catalog/wiki/Roadmap)
 contains information about our high-level goals for future milestones.
 
-We are currently making weekly releases; see the
-[release process](https://github.com/kubernetes-sigs/service-catalog/wiki/Release-Process)
-for more information.
+The release process of Service Catalog is described here: https://github.com/kubernetes-sigs/service-catalog/wiki/Release-Process.
```
Comment (klaudiagrz) [inline README.md]:
```suggestion
The release process of Service Catalog is described [here](https://github.com/kubernetes-sigs/service-catalog/wiki/Release-Process).
```

File: docs/install.md [hunk 4]
```diff
@@ -166,6 +167,13 @@ helm install svc-cat/catalog \
     --name catalog --namespace catalog
 ```
 
+If you want to install older version of service catalog (for instance the  API-server based version from v0.2 branch) append branch name to the chart name:
```
Comment (klaudiagrz) [inline docs/install.md]:
```suggestion
If you want to install an older version of Service Catalog (for instance the API Server-based version from the v0.2 branch) follow the branch name with the name of the chart:
```

File: docs/images/sc-release-process.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (klaudiagrz) [inline docs/images/sc-release-process.svg:1]: It would be nice to start Docker and Helm with uppercase. Also: API Server.

**Review by klaudiagrz — APPROVED** [review]

File: docs/images/sc-release-process.svg (hunk 5 above)
Comment (adamwalach) [reply docs/images/sc-release-process.svg:1]: fixed!

**Pull request closed without merging**